### ISSUE TITLE
The TextBoxHelper.ClearTextButton does not work on DatePicker

### ISSUE THREAD

**Issue by dsaf** [MahApps/MahApps.Metro#2193]

**Comment by punker76** [comment]
it's available in upcoming v1.2.0 (or now in alpha version)

**Comment by dsaf** [comment]
Understood, thank you.

**Comment by dsaf** [comment]
Hi @punker76 I have upgraded to 1.2.0 and then today to 1.2.4 and it seems to be reproducible.

This is the code I am using:

```xaml
<DatePicker SelectedDate="{Binding CreatedOnStart}" mahAppsControls:TextBoxHelper.ClearTextButton="True" mahAppsControls:TextBoxHelper.Watermark="Five Days Ago" />
```

Removing the Watermark makes no difference.

Could you please re-open this issue (unless you cannot reproduce it)? It's not a must-have for me, but it would be nice to have it noted.

**Comment by xxMUROxx** [comment]
memo for me

**Comment by Grizz7778** [comment]
I need or better like to have this, too

**Comment by fennudekugua** [comment]
Hi @punker76 I had the same problem.

This is the code I am using:

 <metro:DateTimePicker metro:TextBoxHelper.Watermark="选择结束时间" 
                        Culture="zh-cn" BorderThickness="0" IsTodayHighlighted="False" 
                        metro:TextBoxHelper.ClearTextButton="True" />

Cleartextbutton still does not work.

**Comment by fennudekugua** [comment]
Hi @punker76 I had the same problem.

This is the code I am using:

```Xaml
<metro:DateTimePicker metro:TextBoxHelper.Watermark="选择结束时间"
Culture="zh-cn" BorderThickness="0" IsTodayHighlighted="False"
metro:TextBoxHelper.ClearTextButton="True" />
```

Cleartextbutton still does not work.

**Comment by fennudekugua** [comment]
> 嗨@ punker76，我有同样的问题。
> 
> 这是我正在使用的代码：
> 
> ```
> < 地铁：的DateTimePicker地铁：TextBoxHelper.Watermark = “选择结束时间” 
> 文化= “ ZH-CN ” 了borderThickness = “ 0 ”  IsTodayHighlighted = “假” 
> 地铁：TextBoxHelper.ClearTextButton = “真” />
> ```
> 
> Cleartextbutton仍然不起作用。

The version I'm using is 1.6.5

**Comment by punker76** [comment]
@fennudekugua It’s available now in current dev (upcoming v2.0)
